### DISCUSSION TITLE
Error after updating nuxt version to 3.10.2 with static project generation

### DISCUSSION THREAD

**Discussion by Misha6787** [nuxt/nuxt#25896] [Questions]
Hello guys, I get an error when generating a static project on nuxt version 3.10.2. Apparently it is somehow connected with the fact that during generation it does not find the folder with fonts, maybe I should somehow change the entry point, for example? In this case, how does it find components if it cannot find fonts?

This error - 
![image](https://github.com/nuxt/nuxt/assets/59843351/057abe85-25c5-4010-9181-550e4d3790e1)
![image](https://github.com/nuxt/nuxt/assets/59843351/d0743907-3c45-4519-b933-7389e69d9962)

For copy - **Error: [404] Cannot find static asset /_nuxt/MYRIADPRO-BOLDCOND.DcN5ydSp.OTF**

And my nuxt config - 
```js
// https://nuxt.com/docs/api/configuration/nuxt-config

export default defineNuxtConfig({
    app: {
        head: {
            viewport: 'width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0',
        }
    },
    css: ['~/assets/default.scss', '~/assets/fonts/fonts.css'],
    ssr: false,
    modules: [
        '@pinia/nuxt',
        '@pinia-plugin-persistedstate/nuxt',
        'nuxt-lodash',
    ],
    plugins: [
        {src: '~/plugins/vMaskPlugin.js'},
        {src: '~/plugins/ymapPlugin.js', mode: 'client'}
    ],
    vite: {
        mode: 'production',
        server: {
            fs: {
                allow: ["C:/node_modules/ufo/dist", "C:/node_modules/defu/dist"]
            }
        }
    },
    build: {
        extend(config, ctx) {
            config.resolve.alias["vue$"] = "vue/dist/vue.esm.js"
        }
    },
    render: {
        bundleRenderer: {
            shouldPrefetch: () => false,
            shouldPreload: () => false,
        },
    }
})

```

**Comment by EddyDezuraud** [comment]
Hey Misha, do you have a link you could share with us to replicate this issue? (e.g., via StackBlitz or CodeSandbox)
What version of Node.js are you using?
